frontend/packages/integration-tests-cypress/cy-command-additions/README.md

```diff
@@ -0,0 +1,11 @@
+# Cypress Custom Commands
+
+Cypress comes with its own API for creating custom commands.  This allows adding custom commands to the Cypress global `cy` variable.
+
+Ex: `cy.createTestProject('test-qbvzv')`
+
+Custom commands work well when you’re needing to describe behavior that’s desirable across all of your tests. Examples would be a cy.setup() or cy.login() or extending your application’s behavior like cy.get('.dropdown').dropdown('Apples'). These are specific to your application and can be used everywhere.
```

Comment:
@dtaylor113 Please add line breaks in these files. That makes it easier to comment on specific changes in PR reviews for instance. 

Not for this PR, but we might consider running `prettier` on our markdown.

Reply:
fixed